### ISSUE TITLE
tracking: Replace Supabase with Radix UI (FUTURE)

### ISSUE THREAD

**Issue by pixelsbyeryc** [open-sauced/insights#338]
## Summary
> **Warning**
> This is a future tracking issue. We're not touching this for some sprints, post-hacktoberfest.

We need to replace all of the Supabase components with Radix UI components.

**Reason:**
- More components available
- Better accessibility features

Reference: https://github.com/open-sauced/insights/issues/318#issuecomment-1242547781

- [x] https://github.com/open-sauced/insights/issues/902
- [x] https://github.com/open-sauced/insights/issues/901

**Comment by bdougie** [comment]
@OgDev-01 can you assist in making na inventory of all the Supabase components.

The PR is blocked on this https://github.com/open-sauced/insights/pull/898

**Comment by OgDev-01** [comment]
I can create issues for them if need be

**Comment by bdougie** [comment]
>IMHO, We could split it into individual issues to avoid too many changes associated with the PR... And also make this a tracking issue for the rest

Make this the tracking issue by editing the Origin Post (issue description) by adding the new issues in a bulleted list using `- [ ]`.

**Comment by takanome-dev** [comment]
@OgDev-01 can we update this issue tracking and open issues for the remaining components to update?

**Comment by bdougie** [comment]
> @OgDev-01 can we update this issue tracking and open issues for the remaining components to update?

Are you not part of the triage team?

**Comment by takanome-dev** [comment]
> Are you not part of the triage team?

I am :sweat_smile: I'm gonna open issues for the rest but I can't edit the issue description.
